cartaya/2/task2.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This is my second task from Leo">
+    <meta name="keywords" content="study,html5,css3,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <link rel="stylesheet" href="style.css">
+    <title>Task#2</title>
+</head>
+ <body>
+    <header>
+        <span id="h1">Intranet</span>
+    </header>
+    <main>
+        <nav>
+            <form action="#" method="get" id="people-search">
+                <div id="lupa" class="color-borde">
+                    <img  class="lupa" src="lupa.png" width="15px" height="15px" alt="search">
+                </div>
+               
+                <input id="search" type="search" name="people-search" placeholder="search" aria-label="search throught site content" class="color-borde">
+                <input id="boton" type="submit" value="Search"> <br /> <br />
+                
+                <input class="selector-azul" checked type="radio" id="last-name" name="people-search" value="last-name" aria-label="last name">

```

Comment:
You need always use a convention to the names of all classes, it should create collisions.